packages/pigeon/pigeons/core_tests.dart

```diff
@@ -96,78 +96,95 @@ abstract class HostIntegrationCoreApi {
 
   /// Returns the passed object, to test serialization and deserialization.
   @ObjCSelector('echoAllTypes:')
+  @SwiftFunction('echo(allTypes:)')
   AllTypes echoAllTypes(AllTypes everything);
 
   /// Returns the passed object, to test serialization and deserialization.
   @ObjCSelector('echoAllNullableTypes:')
+  @SwiftFunction('echo(allNullableTypes:)')
   AllNullableTypes? echoAllNullableTypes(AllNullableTypes? everything);
 
   /// Returns an error, to test error handling.
   void throwError();
 
   /// Returns passed in int.
   @ObjCSelector('echoInt:')
+  @SwiftFunction('echo(int:)')
```

Comment:
Isn't it more idiomatic Swift to make these all echos unnamed arguments? Compare with [the `set` suite on `NSUserDefaults`](https://developer.apple.com/documentation/foundation/userdefaults/1414067-set) for instance.

Reply:
I don't know if it's clear in language.
`NSUserDefaults` is a ObjC class a behave in another way, I thought.
In Swift [documentation](https://docs.swift.org/swift-book/LanguageGuide/Functions.html) there are some examples using `greet(person: String, ...)`, `minMax(array: [Int])`, among others like `someFunction(_ firstParameterName: Int, secondParameterName: Int)`.

But, it could be better, of course and I'm gonna make the changes.

Reply:
Done